job_tracker/static/scripts/contactSearch.js

```diff
@@ -1,19 +1,48 @@
 $(document).ready(function() {
-    let selectEl = document.getElementById("id_application");
-    let j = 0;
+    let contacts = [];
 
-    optionsEl = selectEl.children;
-
-    optionsArray = [...optionsEl];
+    console.log("in js");
 
     $.ajax({
-        url: "/ajax/get_applications/",
         method: "GET",
-        datatype: "json"
-    }).done(res => {
-        for (let i = 1; i < optionsArray.length; i++) {
-            optionsArray[i].text = `${res[j].company}`;
-            j++;
+        url: "/ajax/get_contacts/",
+        success: function(res) {
+            contacts = res;
+        },
+        error: function(res) {
+            console.log("error");
         }
     });
+
+    console.log(contacts);
```

Comment:
can you please delete this console log? Don't want us to miss it and have it be pushed to prod!